Makefile

```diff
@@ -66,19 +66,17 @@ INTEG_TEST_NDC_ROOT    := ./host/ndc
 PROTO_ROOT := proto
 PROTO_FILES = $(shell find ./$(PROTO_ROOT)/internal -name "*.proto")
 PROTO_DIRS = $(sort $(dir $(PROTO_FILES)))
-PROTO_IMPORTS := -I=$(PROTO_ROOT)/internal -I=$(PROTO_ROOT)/api -I=$(GOPATH)/src/github.com/temporalio/gogo-protobuf/protobuf
+PROTO_IMPORTS = -I=$(PROTO_ROOT)/internal -I=$(PROTO_ROOT)/api -I=$(shell cd build && go list -m -f '{{.Dir}}' github.com/temporalio/gogo-protobuf)/protobuf
```

Comment:
This seems bad.. I think all these top-level variables should use `:=`. Otherwise it'll re-run the shell commands every time they're referenced.

Reply:
Unfortunately it is required. Because to return physical path `go list` requires files to exist. But they are downloaded only in a middle of the process which is executed by one single `ci-build` target.

Reply:
Yeah :(  Ok for now but if we can clean this up eventually (e.g. by referring to an actual package) I would like that